File: cpp/program62.cpp
#include<iostream>
using namespace std;

class Pattern
{
    public:
        int iNo;

        Pattern(int X)
        {
            iNo = X;
        }

        void Display()
        {
            char ch = '\0';
            int iCnt = 0;

            for(iCnt = 1,ch = 'a'; iCnt <= iNo; iCnt++,ch++)
            {
                cout<<ch<<"\t"<<iCnt<<"\t";
            }
        }
};

int main()
{
    int iValue = 0;

    cout<<"Enter frequency\n";
    cin>>iValue;

    Pattern *obj = new Pattern(iValue);
    obj->Display();

    return 0;
}

File: java/program62.java
import java.util.*;

class program62
{
    public static void Display(int no)
    {
        int iCnt = 0;
        char ch = ' ';

        for(iCnt = 1,ch = 'A'; iCnt <= no; iCnt++,ch++)
        {
            System.out.print(ch+"\t"+iCnt+"\t");
        }
    }

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter frequency");
        int value = sobj.nextInt();

        Display(value);
    }
}